common/script/index.js

```diff
@@ -1152,6 +1152,53 @@ api.wrap = function(user, main) {
             message: "+1 Gem"
           }, _.pick(user, $w('stats balance'))) : void 0;
         }
+/* buy 5 gems */
+        if (type === 'gems5' && key === 'gem') {
```

Comment:
Wouldn't it be better to adjust the gems logic to take in a quantity instead of having totally separate logic for gem and gem5?